### ISSUE TITLE
Feature Request - More back propagation algorithms

### ISSUE THREAD

**Issue by LYF610400210** [tensorflow/playground#11]
Hello, TensorFlow developers:

This is a feature request. I would like to see how different back propagation algorithms can affect training speed. (Gradient Descent? Rprop? BFGS? I am new to neural networks...)
